building_in_windows.md

```diff
@@ -0,0 +1,73 @@
+# Building the Client in Windows
+These instructions are specific to Windows 10 but should work in earlier versions with a minor bit of tweaking.
+
+## Installing Git
+[Git Download Link](https://git-scm.com/download/win)
+
+Download and install the Git software from the Download Link. Choose to use Git in Git Bash only.
+
+## Installing and Configuring Go
+### Downloading and Installing
+[Go Download Link](https://golang.org/dl/)
+
+Grab the Microsoft Windows installer from the Download Link page. Install via the MSI. After installation, open up a command prompt and run `go version` to ensure a proper response.
+
+### Testing Go
+[Go Getting Started](https://golang.org/doc/install)
```

Comment:
Probably best to link directly to the Windows section: https://golang.org/doc/install#windows.